src/decorators/parameter.ts

```diff
@@ -2,7 +2,7 @@
  * Inject http Body
  *  @param {string} [name] properties name in body object
  */
-export function Body(): Function {
+export function Body(example?: any): Function {
```

Comment:
Don’t use `any` it should be at least a generic with a requirement to extend an object.

Reply:
Ok